includes/controls/groups/typography.php

```diff
@@ -88,17 +86,11 @@ public static function get_type() {
 	protected function init_fields() {
 		$fields = [];
 
-		$default_fonts = Plugin::$instance->kits_manager->get_current_settings( 'default_generic_fonts' );
-
-		if ( $default_fonts ) {
-			$default_fonts = ', ' . $default_fonts;
-		}
-
 		$fields['font_family'] = [
 			'label' => _x( 'Family', 'Typography Control', 'elementor' ),
 			'type' => Controls_Manager::FONT,
 			'default' => '',
-			'selector_value' => 'font-family: "{{VALUE}}"' . $default_fonts . ';',
+			'selector_value' => 'font-family: "{{VALUE}}"' . Plugin::$instance->kits_manager->get_default_fonts() . ';',
```

Comment:
Move the comma from `get_default_fonts` to here.
```suggestion
			'selector_value' => 'font-family: "{{VALUE}}, "' . Plugin::$instance->kits_manager->get_default_fonts() . ';',
```